tests/fsharp/TypeProviderTests.fs

```diff
@@ -32,13 +32,17 @@ let FSC_OPTIMIZED = FSC_NETFX (true, false)
 let FSI = FSI_NETFX
 #endif
 
-let inline getTestsDirectory dir = getTestsDirectory __SOURCE_DIRECTORY__ dir
-let testConfig = getTestsDirectory >> testConfig
+let copyHelloWorld cfgDirectory =
```

Comment:
Shouldn't this be done via the new helper copy dir function?

Reply:
I think this is a special case.  "helloworld" folder comes from a level up, i.e. ".."
This extra directory level is accommodated for in the general mechanism to not complicate things.

Reply:
Alright then.